definitions/push-application-snapshot/push-application-snapshot.yaml

```
@@ -0,0 +1,49 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: push-application-snapshot-task
```

We are not using the `-task` suffix.
```suggestion
  name: push-application-snapshot
```